### ISSUE TITLE
Discussion: link-name changes

### ISSUE THREAD

**Issue by jkodu** [auto-wcag/auto-wcag#267]
The following test case currently `fails`.

Scenario: Not exposed to assistive technologies.
```html
<a href="http://www.w3.org/WAI" aria-hidden="true"><img src="#" /></a>
```

Believe, this should fail unless it's not focusable.
This means a rule change is required for rule `SC2-4-4-link-has-name`

**Comment by annethyme** [comment]
Decision on Auto-WCAG September call: 
Update definition of Exposed to Assistive Technologies, https://github.com/auto-wcag/auto-wcag/issues/108 - maybe with a section about Accessibility Support. 
Questions is whether AT not respecting aria-hidden for focusable elements is Accessibility Support issue or valid case for rules,

**Comment by Brynanders** [comment]
Resources
Definition of `aria-hidden` https://www.w3.org/TR/wai-aria-1.1/#aria-hidden
Definition of 'hidden' https://www.w3.org/TR/wai-aria-1.1/#dfn-hidden

**Comment by jkodu** [comment]
PR - https://github.com/auto-wcag/auto-wcag/pull/269
